src/main.rs

```diff
@@ -4,16 +4,76 @@ use cairo_foundry::cli::{
 	formatter::{self, Formatter},
 };
 use clap::Parser;
+use confy;
 use log::error;
+use serde_derive::{Deserialize, Serialize};
+
+const CONFIGURATION_FILENAME: &str = "config.toml";
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+struct Config {
+	args: String,
+}
+
+/// Tries to retrieve the Args from the CLI. If the user left the CLI empty, tries to
+/// retrieve the args from the configuration file.
+fn get_args() -> cli::Args {
+	// Try to parse arguments from stdin
+	let args_from_stdin: Result<cli::Args, clap::Error> = cli::Args::try_parse();
+
+	match args_from_stdin {
+		Err(stdin_err) => {
+			// Rust trick to help with type system
+			let stdin_err: clap::Error = stdin_err;
+
+			match stdin_err.kind() {
+				// This means the user left the CLI empty. We can try parsing the configuration file
+				clap::ErrorKind::DisplayHelpOnMissingArgumentOrSubcommand => {
```

Comment:
You are only handling one case, so maybe an `if let / else` would be more expressive there

Reply:
https://github.com/onlydustxyz/cairo-foundry/commit/ae3a980947f4ee64a932610637d6c54aadd24afa